test/ASTGen/verify-parse.swift

```diff
@@ -1,5 +1,7 @@
 // RUN: %target-run-simple-swift(-enable-experimental-feature SwiftParser -enable-experimental-feature ParserASTGen)
 
+// REQUIRES: executable_test
```

Comment:
Most `target-run-simple-swift` tests are marked `executable_test` because they must be run as native executables. That's why this new test fails on the community Android CI, which only runs cross-compiled tests.